Xplat/src/main/java/vazkii/botania/api/BotaniaAPI.java

```diff
@@ -220,4 +221,12 @@ default void sparkleFX(Level world, double x, double y, double z, float r, float
 	default void registerCorporeaNodeDetector(CorporeaNodeDetector detector) {
 
 	}
+
+	default boolean isInGaiaArena(Entity entity) {
```

Comment:
I would omit this to keep the api lean. Callers can write the get calls themselves.